modular_skyrat/modules/company_imports/code/armament_datums/kahraman_industries.dm

```diff
@@ -5,47 +5,31 @@
 // Mining PPE, SEVAs and hardhats, have you passed your OSHA inspection today?
 
 /datum/armament_entry/company_import/kahraman/ppe
-	subcategory = "OSHA Certified Protective Equipment"
-
-/datum/armament_entry/company_import/kahraman/ppe/hardhat
-	item_type = /obj/item/clothing/head/utility/hardhat/orange
-	lower_cost = CARGO_CRATE_VALUE * 0.25
-	upper_cost = CARGO_CRATE_VALUE
+	subcategory = "FOHSA Certified Protective Equipment"
```

Comment:
Why the change?

Reply:
OSHA is an american only organisation where as FOHSA (made up on the spot) for Frontier Occupational Health & Safety Administration is a bit more applicable to the setting.